runtime-library/README.md

```diff
@@ -53,24 +53,34 @@ header files.
 
 ## Build for iOS
 
-Follow [this step](https://iree-org.github.io/iree/building-from-source/ios/#build-the-iree-compiler-for-the-host) to build the IREE compiler for macOS.
-Let's say the CMake build directory is `$HOME/w/iree/build`. To configure and build this project for the iOS Simulator, we could use the following commands.
+Download Xcode from the App Store.  Stick with the Python that comes with Xcode.
 
+Don't try to be clever by downloading a specific version of Xcode from https://developer.apple.com/download or installing Python from python.org. Homebrew may download Xcode Command Line Tools, however this is inadequate to develop IREE for iOS, tvOS, and watchOS.
+
+Install [Homebrew](https://brew.sh/) after installing Xcode.
+
+Install build tools.
+
+```bash
+brew install cmake ninja tree
 ```
-cmake -S . -B build-ios-sim -GNinja \
-  -DCMAKE_SYSTEM_NAME=iOS \
-  -DCMAKE_OSX_SYSROOT=$(xcodebuild -version -sdk iphonesimulator Path) \
-  -DCMAKE_OSX_ARCHITECTURES=arm64 \
-  -DCMAKE_SYSTEM_PROCESSOR=arm64 \
-  -DCMAKE_OSX_DEPLOYMENT_TARGET=11.0 \
-  -DCMAKE_IOS_INSTALL_COMBINED=YES \
-  -DIREE_HOST_BIN_DIR="$HOME/w/iree-build/install/bin" \
-  -DCMAKE_INSTALL_PREFIX=../build-ios-sim/install \
-  -DIREE_BUILD_COMPILER=OFF
-  
-cmake --build build-ios-sim
+
+Git-clone IREE to a directory side-by-side with this repository.
```

Comment:
Nit: new line to enable markdown highlighting to work

Reply:
Added a new line and ran mdformat.